ui/src/components/Tasks/Helical.jsx

```diff
@@ -73,6 +75,7 @@ class Helical extends React.Component {
     resetLastUsedParameters(this);
   }
 
+  // TODO: add widgets for entering space group and unit cell parameters
```

Comment:
Remove this line.

The widgets for space group and cell params are added, thus TODO have been DONE :)